chromedp_test.go

```diff
@@ -261,6 +261,12 @@ func TestCancelError(t *testing.T) {
 		t.Fatalf("expected a nil error, got %v", err)
 	}
 
+	allocCtx, allocCancel := NewExecAllocator(context.Background(), allocOpts...)
+	defer allocCancel()
```

Comment:
I think we don't need a new `allocCtx`. Just use the one that already exists.

Reply:
You're right, I didn't realise it was already defined.